test/matrixvariates.jl

```diff
@@ -413,7 +413,7 @@ end
 function test_special(dist::Type{LKJ})
     @testset "LKJ mode" begin
         @test mode(LKJ(5, 1.5)) == mean(LKJ(5, 1.5))
-        @test_throws ArgumentError mode( LKJ(5, 0.5) )
+        @test_throws DomainError mode( LKJ(5, 0.5) )
```

Comment:
Is this change considered breaking?

Reply:
No, not according to ColPrac: https://colprac.sciml.ai/#changes-that-are-not-considered-breaking